### PR TITLE
Start the sync of the fedora35 python 3.10 solvers

### PR DESCRIPTION
**Pull request by harshad16**
Start the sync of the fedora35 python 3.10 solvers
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Fixes: https://github.com/thoth-station/thoth-application/issues/2479
Related-to: https://github.com/thoth-station/s2i-thoth/issues/231

## Description

start the solver execution for the f35-py3.10 based solvers.

### CODE REVIEW

**Comment by goern** [comment]
/hold
the container image tag is missing, and the build is blocked by https://github.com/thoth-station/s2i-thoth/issues/242

**Comment by harshad16** [comment]
> the container image tag is missing, and the build is blocked by [thoth-station/s2i-thoth#242](https://github.com/thoth-station/s2i-thoth/issues/242)

good catch , i had tagged a wrong image.
Ready for review again.
/unhold
https://quay.io/repository/thoth-station/solver-fedora-35-py310?tab=tags

File: core/overlays/test/configmaps.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ metadata:
   name: solvers
 data:
   solvers: |
-    solver-rhel-8-py36
```
Comment (fridex) [inline core/overlays/test/configmaps.yaml:38]: We could keep the old solver for py36 here. Test environment is smaller so we don't need to trigger ingestion there.

**Review by fridex — COMMENTED** [review]
One comment, otherwise LGTM! 💯

File: core/overlays/test/configmaps.yaml (hunk 1 above)
Comment (harshad16) [reply core/overlays/test/configmaps.yaml:38]: ack, thanks for pointing it out.

**Comment by harshad16** [comment]
Merging this as already reviewed by reviwers
/approve

**Pull request merged**